### ISSUE TITLE
Accuracy on Terabyte Dataset

### ISSUE THREAD

**Issue by thanhtuansnu** [facebookresearch/dlrm#200]
Would it be possible to publish the hyper-parameters used to match with MLPerf target AUC?
I can find ones that used for the case where the full dataset is used but I wonder if you have the hyperparameters when --data-sub-sample-rate=0.875 is used?

**Comment by mnaumovfb** [comment]
The MLPerf/MLCommons target AUC can be achieved by running the shell script [run_and_time.sh](https://github.com/facebookresearch/dlrm/blob/master/bench/run_and_time.sh)

The flag `--data-sub-sample-rate=0.875` will drop 7/8 = 0.875 of the training samples. It would have resulted in a completely different AUC targets and it does not make sense to compare it with the MLPerf/MLCommons runs.

**Comment by thanhtuansnu** [comment]
Thanks for clarification!
